docs/source/04_user_guide/08_advanced_io.md

```diff
@@ -52,16 +52,16 @@ In order to enable versioning, you need to update the `catalog.yml` config file
 An example dataset could look similar to the below:
 
 ```python
-from pathlib import Path
+from pathlib import PurePath
```

Comment:
We normally use `PurePosixPath` in concrete datasets 🤔 but maybe `PurePath` is also ok
```suggestion
from pathlib import PurePosixPath
```